bin/addRedirectURItoConnect.ts

```diff
@@ -0,0 +1,54 @@
+import { updateApplication } from "../lib/commands/updateApplication";
+import { getApplication } from "../lib/queries/getApplication";
+import { config } from "../src/config";
+
+const addingRedirectURIToConnect = async (): Promise<any> => {
+  try {
+    if (process.env.GITHUB_CONTEXT_EVENT === undefined)
+      throw new Error("githubContextEvent is undefined");
+
+    const githubActionsContext = JSON.parse(process.env.GITHUB_CONTEXT_EVENT);
+
+    if (githubActionsContext.deployment_status === undefined)
+      throw new Error("deploymentStatus is undefined");
+
+    const vercelDeployment = githubActionsContext.deployment_status;
+
+    if (vercelDeployment.state === "success") {
+      const testApp = await getApplication(config.connectAccountTestAppId).then(
```

Comment:
I like what you did with your truthly checks on the env vars below. Now that I think about it, I was probably wrong when I told you to add this to the config object (because it only concerns GitHubs Actions env). What do you think?

Reply:
Yep, since this env var is only used in GH actions, I think there's no need to add it on project's config object (and it may be a bit confusing to see it in config obj, but used nowhere in the projects files). But I may be wrong. :)

Reply:
You are absolutely right :)